electrification.mss

```diff
@@ -169,6 +173,20 @@
       }
     }
 
+    [frequency!=0][voltage<15000] {
+       line-color: #97FF2F;
+    }
+
+    [frequency!=0][voltage>=15000][voltage<20000],
```

Comment:
Not 100% sure, but I think somethiong like
```suggestion
    [frequency!=0][frequency!=16.67][frequency!=16.7][voltage>=15000][voltage<20000],
```
is needed in order to not render the whole 15kV/16.xx area of Europe in `#F1F100`.